tests/em/nsem/inversion/test_Problem3D_Derivs.py

```diff
@@ -91,35 +91,6 @@ def fun(x):
     return tests.check_derivative(fun, m, num=3, plotIt=False, eps=FLR)
 
 
-def DerivProjfieldsTest(inputSetup, comp="All", freq=False):
```

Comment:
this wasn't run, and with all of the development work that @dccowan is doing in #1454, a lot more thorough testing will be added to the NSEM module, so I think we are okay removing this